### PR TITLE
Add an RDS Aurora MySQL 1 Cluster instance that does not inherit configuration from the cluster.

### PR DESCRIPTION
**Pull request by choilmto**
1. `final_snapshot_identifier` is an optional argument for the aurora resource object. However, upon destroying the resource, [it seems necessary](https://stackoverflow.com/questions/50930470/terraform-error-rds-cluster-finalsnapshotidentifier-is-required-when-a-final-s) to either provide a name for the `final_snapshot_identifier` or by setting `skip_final_snapshot` to `true`.

2. Can someone unpack [the note in the documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster) about arguments in terraform including the username and password being stored in the raw state as plain-text.

### CODE REVIEW

**Comment by choilmto** [comment]
Instead of adding a solution for securing the login credentials, I've added some documentation in the terraform file to keep the module simple.

File: aws/aws_aurora/cluster/main.tf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Documentation: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster
+resource "aws_rds_cluster" "changeme_aws_aurora_cluster" {
+  # The login credentials are stored as plain-text in the state files
+  # https://www.terraform.io/docs/language/state/sensitive-data.html
+  # For security, use a secrets manager
+  # https://registry.terraform.io/modules/yurymkomarov/rds-aurora-cluster/aws/latest#secretsmanager_secret
+  master_username     = var.changeme_aws_aurora_username
+  master_password     = var.changeme_aws_aurora_password
+  skip_final_snapshot = true
+}
```
Comment (ttarczynski) [inline aws/aws_aurora/cluster/main.tf:46]: This resource caused a error with `terraform apply` done by CI:
- https://github.com/ContainerSolutions/terraform-examples/runs/3229040694?check_suite_focus=true

```
Plan: 1 to add, 0 to change, 0 to destroy.
aws_rds_cluster.changeme_aws_aurora_cluster: Creating...
╷
│ Error: error creating RDS cluster: AccessDenied: User: arn:aws:iam::684157722011:user/terraform-examples is not authorized to perform: rds:CreateDBCluster on resource: arn:aws:rds:us-east-1:684157722011:cluster:tf-20210803085818971100000001
│ 	status code: 403, request id: d380fcf2-c6e7-47c7-ae23-dacc3f0d6782
│ 
│   with aws_rds_cluster.changeme_aws_aurora_cluster,
│   on main.tf line 38, in resource "aws_rds_cluster" "changeme_aws_aurora_cluster":
│   38: resource "aws_rds_cluster" "changeme_aws_aurora_cluster" ***
│ 
╵
/home/runner/work/terraform-examples/terraform-examples/aws/aws_aurora/cluster
```

Seems to be related to some missing permissions.

**Comment by ttarczynski** [comment]
> Instead of adding a solution for securing the login credentials, I've added some documentation in the terraform file to keep the module simple.

It makes sense that way. I production setup we don't want to pass secrets this way because they are then stored in the tfstate file. But for our examples it's what we want (for simplicity).

**Review by ttarczynski — APPROVED** [review]
LGTM
After fixing the permissions error (Ian added `RDSAdmin` role) CI works OK.

**Pull request merged**